pkg/declarative/v2/raw_render.go

```diff
@@ -0,0 +1,46 @@
+package v2
+
+import (
+	"context"
+	"os"
+
+	"k8s.io/client-go/tools/record"
+)
+
+func NewRawRenderer(
+	spec *Spec,
+	options *Options,
+) Renderer {
+	return &RawRenderer{
+		EventRecorder: options.EventRecorder,
+		Path:          spec.Path,
+	}
+}
+
+type RawRenderer struct {
+	record.EventRecorder
+	Path string
+}
+
+func (r *RawRenderer) Initialize(_ Object) error {
```

Comment:
why there is no `Initialize` and `EnsurePrerequisites` for raw render? is it for implement later?

Reply:
There is no need for any initialisation or prerequisites since it only reads a yaml file and parses this.

Reply:
OK, I understood for `EnsurePrerequisites`. but for`Initialize`, do we need some similar check like helm initialization, check exists for chart path. I mean the raw render, the manifests.yaml file which it reads is saved somewhere, right?

Reply:
The manifest.yaml is used as is since it does not need to render anything (there are no values or things like that). Thus you also don't need to initialise anything that would change that manifest. Thus the render will actually just access the Path, nothing else.

Reply:
Yes, but the exists of this manifest.yaml should be check, or? I mean what if the file get deleted or not generated due to some file permission problem. like what in helm render does: 
```
	loadedChart, err := loader.Load(h.chartPath)

Reply:
If the file does not exist it will error out in the reconcile loop. In the same way it will error out on RawRender. You can try it side by side if you want to see it in action but you will not notice a difference.

Reply:
yes, in this case check existence is a bit redundant.